orderapi/src/test/java/fi/hel/verkkokauppa/order/service/accounting/AccountingSearchServiceTest.java

```diff
@@ -0,0 +1,127 @@
+package fi.hel.verkkokauppa.order.service.accounting;
+
+import fi.hel.verkkokauppa.order.model.Order;
+import fi.hel.verkkokauppa.order.model.accounting.AccountingExportData;
+import fi.hel.verkkokauppa.order.test.utils.SearchAfterServiceTestUtils;
+import fi.hel.verkkokauppa.order.testing.annotations.RunIfProfile;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.core.env.Environment;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+
+@RunWith(SpringJUnit4ClassRunner.class )
+@SpringBootTest
+@RunIfProfile(profile = "local")
+@TestPropertySource(properties = { "elasticsearch.search-after-page-size = 7" })
+@ComponentScan("fi.hel.verkkokauppa.order.service.elasticsearch")
+@Slf4j
+public class AccountingSearchServiceTest extends SearchAfterServiceTestUtils {
+
+    @Autowired
+    private Environment env;
+
+    @Autowired
+    AccountingSearchService searchServiceToTest;
+
+    @Autowired
+    AccountingExportDataService accountingExportDataService;
+
+
+    @Test
+    public void whenGetNotExportedAccountingExportDataAndNoHits_thenSuccess() throws Exception {
+        log.info("running whenGetNotExportedAccountingExportDataAndNoHits_thenSuccess");
+        clearAccountingExportData();

```

Comment:
Check if this can be done with tearDown [example](https://github.com/City-of-Helsinki/verkkokauppa-core/blob/1741e834cd52a28d49e85a496593a59a89d96e44/orderapi/src/test/java/fi/hel/verkkokauppa/order/api/OrderControllerTest.java#L59)